src/main/java/io/xdb/core/process/Process.java

```diff
@@ -20,6 +21,16 @@ default ProcessOptions getOptions() {
      * @return the state schema of all {@link io.xdb.core.state.AsyncState} defined for this process.
      */
     default StateSchema getStateSchema() {
-        return StateSchema.builder().build();
+        return StateSchema.noStartingState();
+    }
+
+    /**
+     * The default persistence schema to load in each {@link io.xdb.core.state.AsyncState}.
+     * Users can override this loading setting in each state's {@link io.xdb.core.state.AsyncStateOptions}.
+     *
+     * @return the persistence schema defined for this process.
+     */
+    default PersistenceSchema getPersistenceSchema() {
+        return PersistenceSchema.EMPTY();
```

Comment:
@longquanzheng I feel there is little needs to define persistence schema for validating. Instead, we can define a default loading way in Process for each AsyncState. WDYT?